### DISCUSSION TITLE
Spotify API for Artists, Albums and Tracks

### DISCUSSION THREAD

**Discussion by NoelJacob** [airbytehq/airbyte#36110] [New Connector Request]
## Tell us about the new connector you’d like to have
I'd like to see the Spotify API's Albums, Artists and Tracks information endpoints. Complete list of endpoints [here](https://developer.spotify.com/documentation/web-api/reference).

## Describe the context around this new connector
API user will need API keys from Spotify.

## Describe the alternative you are considering or using
Get Spotify API info manually.

## Are you willing to submit a PR?
Yes! with the Low-Code CDK.

**Comment by marcosmarxm** [comment]
Nice, let me know if you need anything!

**Comment by NoelJacob** [comment]
Ok I'm on it... I've filled contributor reward form but haven't been added to Slack.

**Comment by btkcodedev** [comment]
@jrolom I would like to work on this issue :)
I have premium account of Spotify to get the credentials

[a comment by an automated account is not shown]
